english/ninja/ninja.md

```diff
@@ -17,3 +17,20 @@ I'm like a wild animal in the corner
 Waiting for the break of dawn
 Trying to get through the night
 Just a man with the will to survive
+
+
+After that episode,
+
+You feel like your singing has reached its ultimate
+ability, and never in your life will you reach this level
+of skill or readiness again...
+
+You decide to do the unthinkable...
+
+[Try out Karaoke](../karaoke/none.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.